packages/cloudrun/install.mjs

```diff
@@ -0,0 +1,201 @@
+// this will be executed with node
+import {execSync} from 'child_process';
+import readline from 'readline';
+
+execSync(
+	'echo "\n\n\u001b[32;1m◀️◀️◀️ GCP project setup for Remotion Cloud Run ▶️▶️▶️\n\n\u001b[0m"',
+	{stdio: 'inherit'}
+);
+
+/****************************************
+ * Prompt user for Remotion version
+ ****************************************/
+
+function versionPrompt() {
+	return new Promise((resolve) => {
+		// regex to ensure remotionVersion is in semver format
+		const semverRegex = new RegExp(
+			/^v?(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/,
+			'i'
+		);
+
+		const rl = readline.createInterface({
+			input: process.stdin,
+			output: process.stdout,
+		});
+
+		rl.question(
+			'What version of Remotion do you want to use (format: 1.0.0)? \u001b[34;1m',
```

Comment:
Let's definitely default it to latest (getLatestRemotionVersion() function could be copied from somewhere else